public/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="pl">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/index.css" />
+    <title>Form JS Challenge</title>
+  </head>
+  <body>
+    <form class="form">
+      <div class="form__field">
+        <label class="form__label">Haslo 1</label>
+        <input type="text" class="form__input" />
+      </div>
+      <div class="form__field">
+        <label class="form__label">Haslo 2</label>
+        <input type="text" class="form__input" />
+      </div>
+      <div class="form__field">
+        <label class="form__label">Haslo 3</label>
+        <input type="text" class="form__input" />
+        <button class="btn__field ">
+          <img src="css/icon/close.svg" height="30" width="30" />
+        </button>
+      </div>
+      <div class="form__field">
+        <label class="form__label">Haslo 4</label>
+        <input type="text" class="form__input" />
+        <button class="btn__field ">
+          <img src="css/icon/close.svg" height="30" width="30" />
+        </button>
+      </div>
+      <button class="btn btn-blue">Dodaj</button>
```

Comment:
jak chcesz w jakiś sposób sterować pozycją buttona - to opakuj go w jakiegoś diva, który będzie za to odpowiedzialny

Reply:
Obecnie chyba każdy z tych butonie będzie ci submitowac forma bo tyle submit jest domyślny, daj im type button. Albo steruj w jsie